Source/BoundaryConditions/PEC_Insulator.H

```diff
@@ -91,18 +91,12 @@ public:
      * \param[in]     a_Fz_x_lo the parser for the tangential field at the boundary
      * \param[in]     a_Fy_x_hi the parser for the tangential field at the boundary
      * \param[in]     a_Fz_x_hi the parser for the tangential field at the boundary
-     * \param[in]     set_F_y_lo whether the tangential field at the boundary was specified
-     * \param[in]     set_F_y_hi whether the tangential field at the boundary was specified
-     * \param[in]     a_Fx_y_lo the parser for the tangential field at the boundary
-     * \param[in]     a_Fz_y_lo the parser for the tangential field at the boundary
-     * \param[in]     a_Fx_y_hi the parser for the tangential field at the boundary
-     * \param[in]     a_Fz_y_hi the parser for the tangential field at the boundary
-     * \param[in]     set_F_z_lo whether the tangential field at the boundary was specified
-     * \param[in]     set_F_z_hi whether the tangential field at the boundary was specified
-     * \param[in]     a_Fx_z_lo the parser for the tangential field at the boundary
-     * \param[in]     a_Fy_z_lo the parser for the tangential field at the boundary
-     * \param[in]     a_Fx_z_hi the parser for the tangential field at the boundary
-     * \param[in]     a_Fy_z_hi the parser for the tangential field at the boundary
+     * \\param[in]     set_F_z_lo whether the tangential field at the boundary was specified
```

Comment:
Are the double `\\` intentional here?

Reply:
Yes, strangely enough Doxygen was not finding the parameters in the function signature. 

These are inside some `#if defined` conditions, e.g., `#if defined(WARPX_ZINDEX)`, and I'm not sure how Doxygen treats those. 

But I definitely had to "remove" them from the list of parameters in order for Doxygen to build without warnings. 

Since I also wanted to keep them documented, I used the `\\` trick, but it could have been any syntax that could escape Doxygen.

Reply:
What this function tells me though is that it should have an overloaded API for the different geometries...